### ISSUE TITLE
Feature request: Mention bridge as attribute of highway

### ISSUE THREAD

**Issue by goldfndr** [streetcomplete/StreetComplete#5792]
**Use case**
There are many places where a bridge will overlap another highway at a sharp angle or even coincide with the same angle for a great distance. The rendering of bridges uses rounded caps but, from what I can tell, not for transitions within the bridge (e.g. lane count change). It can be difficult to distinguish which piece of road is a bridge and which is at ground level, especially within a downtown area with limited right of way for expansion.

 For example, [a motorway bridge](https://www.openstreetmap.org/way/402084473) overlapping [ a ground-level motorway](https://www.openstreetmap.org/way/4755256).

**Proposed Solution**
In #1326 / [b6d286a](https://github.com/streetcomplete/StreetComplete/commit/b6d286ae3da9571297a8d83c58d6a9544b13be57) a tunnel attribute was added for subject roads; improved in #3870. A bridge attribute could similarly be displayed as appropriate.

**Comment by goldfndr** [comment]
I took a look through the quests (including SCEE's) and I'm not spotting one in which location=overhead or location=overground  or location=roof might apply that didn't already have **level**; quest objects seem to be at (or, in the case of barriers like turnstiles and gates, at least attached to) ground level, even the height quests are measured from ground. I didn't spot a pipe/cable color/substance/material quest, and a future power pole attachment quest oughtn't need it (but other power attachments might). Closest I saw was possibly SCEE's color quest for a `building:part` that lacked `level` but that seems more likely bad tagging. Is a stile ever fully elevated?

So, for now, I'd say something like:
```val bridge = tags["bridge"]
 if (bridge != null && bridge != "no") {
     return resources.getString(R.string.bridge)
}```
